### ISSUE TITLE
vglconnect uses ipv6, but vglrun can't connect back

### ISSUE THREAD

**Issue by thomasschaeferm** [VirtualGL/virtualgl#215]
`vglconnect -ipv6 servername`

does work as expected, it connects to the server and vglcient listens to ipv6 and ipv4

but

```
vglrun glxgears
[VGL] NOTICE: Automatically setting VGL_CLIENT environment variable to
[VGL]    1, the IP address of your SSH client.
```

does complete nonsense and the connection back to port 4242 fails

even vglconnect without `-ipv6` prefers ipv6 (but opens the vglclient socket ipv4-only)

The only way to get it running is to use ipv4-literals instead of the server name.
But that is only a very ugly workaround.

versions:

2.5.2 at the gpu server
2.6.5 at the client (user's pc)

**Comment by dcommander** [comment]
2.5.2 had some known bugs with IPv6 connections.  The 2.6 release series was created mostly because I reorganized a lot of things in the code, not because it contains any significant new features relative to 2.5.x.  Thus, 2.6.5 can be considered a bug-fix release relative to 2.5.2, and one of the bugs it should fix is this one.  Please upgrade the server to 2.6.5 and retest.

**Comment by thomasschaeferm** [comment]
2.6.5 is ok.
